internal/testing/e2e/backfill_test.go

```diff
@@ -177,31 +178,23 @@ func TestProposedBackfillCreate(t *testing.T) {
 		Config:  om.MMFConfigGRPC(),
 		Profile: &pb.MatchProfile{},
 	})
-	require.Nil(t, err)
+	require.NoError(t, err)
 
 	resp, err := stream.Recv()
-	require.Nil(t, err)
+	require.NoError(t, err)
 	bfID := resp.Match.Backfill.Id
 
 	resp, err = stream.Recv()
 	require.Nil(t, resp)
 	require.Equal(t, io.EOF, err)
 
 	actual, err := om.Frontend().GetBackfill(ctx, &pb.GetBackfillRequest{BackfillId: bfID})
-	require.Nil(t, err)
+	require.NoError(t, err)
 	require.NotNil(t, actual)
 
 	b.Id = actual.Id
 	b.CreateTime = actual.CreateTime
 	require.True(t, proto.Equal(b, actual))
-
-	client, err := om.Query().QueryTickets(ctx, &pb.QueryTicketsRequest{Pool: &pb.Pool{
```

Comment:
curious why this blurb was removed? seems unrelated to this change itself?

Reply:
Had issues with this test, agreed with @yeukovichd to remove these lines at some point, but this test is updated in #1318 so I will return this code back.

Reply:
Will this be returned back in this same PR? Please resolve the comment when it is reverted back. Thanks!

Reply:
This test is already updated in #1318